### ISSUE TITLE
MoneyExtension - number_format() expects parameter 1 to be float, string given 

### ISSUE THREAD

**Issue by KadelaKuba** [shopsys/shopsys#1768]
### What is happening

PHP function for formatting number `number_format` expects parameter to be float, but in method `moneyFormatFilter` in class `MoneyExtension` is given string. 

```
public function moneyFormatFilter( ) {
  $moneyString = $money->getAmount();

   if ($decimal === null) {
       $decimal = $this->getNumberOfDecimalPlaces($moneyString);
   }

  return number_format($moneyString, $decimal, $decimalPoint, $thousandsSeparator);
}
```

When you have `declare(strict_types=1); ` in extended class `MoneyExtension` it cause Type error. 

### Expected result

Give float typed parameter to function `number_format()`

**Comment by pesektomas** [comment]
Hi @KadelaKuba, thank you for reported this issue. This is very `nice` bug :smile:

**Comment by KadelaKuba** [comment]
Remove the stale.

**Comment by s3tezsky** [comment]
Thanks for reporting. This issue was already fixed in #1922! 🎉
